gen/archs.xml

```diff
@@ -96,7 +96,6 @@
   </check>
   <flag compiler="gnu">-mmmx</flag>
   <flag compiler="clang">-mmmx</flag>
-  <flag compiler="msvc">/arch:SSE</flag>
```

Comment:
MSVC fully removed MMX support from their compilers therefore there is no reason to check for that